src/common/command.rs

```diff
@@ -25,7 +25,7 @@ impl Display for CommandAndArguments {
             .map(|a| a.escape_default().collect::<String>())
             .collect::<Vec<_>>()
             .join(" ");
-        write!(f, "{} {}", cmd, args)
+        write!(f, "{cmd} {args}")
```

Comment:
Ah, this has returned to clippy? 😁  I remember this being briefly in it in 2023 and then they removed it again. Keeping us busy...